src/wp-admin/includes/schema.php

```diff
@@ -1209,38 +1209,14 @@ function populate_network_meta( $network_id, array $meta = array() ) {
 --The Team @ SITE_NAME'
 	);
 
-	$misc_exts        = array(
-		// Images.
-		'jpg',
-		'jpeg',
-		'png',
-		'gif',
-		'webp',
-		// Video.
-		'mov',
-		'avi',
-		'mpg',
-		'3gp',
-		'3g2',
-		// "audio".
-		'midi',
-		'mid',
-		// Miscellaneous.
-		'pdf',
-		'doc',
-		'ppt',
-		'odt',
-		'pptx',
-		'docx',
-		'pps',
-		'ppsx',
-		'xls',
-		'xlsx',
-		'key',
-	);
-	$audio_exts       = wp_get_audio_extensions();
-	$video_exts       = wp_get_video_extensions();
-	$upload_filetypes = array_unique( array_merge( $misc_exts, $audio_exts, $video_exts ) );
+	$allowed_file_types = array();
+	$all_mime_types     = get_allowed_mime_types();
+
+	foreach ( $all_mime_types as $ext => $mime ) {
+		array_push( $allowed_file_types, ...explode( '|', $ext ) );
+	}
+	sort( $allowed_file_types );
+	$upload_filetypes = array_unique( $allowed_file_types );
```

Comment:
Similarly, do we need `array_unique` here? `get_allowed_mime_types` is expected to return a list with unique types already and we don't do this elsewhere.

Reply:
There is a possibility here that someone may have filter the value and adding a repeated mime type / ext.

Reply:
sure I understand it is a defensive call to cover these cases, but everywhere else in core when we call `get_allowed_mime_types` we just use the returned values without first normalizing them in this way. What are the potential consequences of having an erroneous duplicate value?

Reply:
Just trying to be overly careful here.

Reply:
It worth noting this has been done  - https://github.com/WordPress/wordpress-develop/blob/2fc2afb107785f0197e1324fa380e3a0b382cdd1/src/wp-admin/includes/schema.php#L1243